wallet/src/manifest/manifest.json

```diff
@@ -1,7 +1,7 @@
 {
-    "name": "AUTO_UPDATED",
-    "description": "AUTO_UPDATED",
-    "version": "AUTO_UPDATED",
+    "name": "Sui Wallet Extension",
```

Comment:
These fields are automatically updated [here](https://github.com/MystenLabs/sui/blob/main/wallet/configs/webpack/webpack.config.common.ts#L154) using the values from the [package.json](https://github.com/MystenLabs/sui/blob/main/wallet/package.json) and the [APP_NAME](https://github.com/MystenLabs/sui/blob/main/wallet/configs/webpack/webpack.config.common.ts#L16). So we should change those values instead of the manifest itself in order for this to work.

Reply:
Got it thanks for the info!